### PR TITLE
Add scripts to run end-to-end workflows locally

### PR DESCRIPTION
**Pull request by aethr**
## Description

Both the daily and the monthly run from om-infra transcribed to run on locally built docker containers.

This should facilitate easier testing of changes on small-scale domains by building the containers locally, and running against code on the developer machine.

## Checklist

Please confirm that this pull request has done the following:

- [ ] Tests added
- [x] Documentation added (where applicable)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

File: scripts/docker-build-all.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)
+
+OPENMETHANE_PATH=${OPENMETHANE_PATH:-"$SCRIPT_DIR/.."}
+OPENMETHANE_PRIOR_PATH=${OPENMETHANE_PRIOR_PATH:-"$OPENMETHANE_PATH/../openmethane-prior"}
+SETUP_WRF_PATH=${SETUP_WRF_PATH:-"$OPENMETHANE_PATH/../setup-wrf"}
```
Comment (aethr) [inline scripts/docker-build-all.sh:7]: This script assumes that `openmethane-prior` and `setup-wrf` are checked out in sibling folders next to `openmethane`, with the same name as the (current) git repo. If your setup looks a bit different, you can manually supply the paths via:

```shell
OPENMETHANE_PRIOR_PATH=/path/to/prior SETUP_WRF_PATH=/path/to/setup_wrf scripts/docker-e2e-daily.sh
```

**Comment by aethr** [comment]
@prayner I've completed a local run of both daily and monthly workflows with these scripts, but I fear there may still be some gremlins. I don't have the experience to understand whether some of the warnings and errors are expected, or whether they represent failures.

I was able to get a `final_solution.ncf` from the end of the monthly run, where I was able to check for the `openmethane_version` attribute, so for my purposes this was a success.

```
$ ncdump 2022-07-22/archive_Pert/202207_test/final_solution.ncf
netcdf final_solution {
dimensions:
	ROW = 5 ;
	COL = 5 ;

// global attributes:
		:SDATE = 2022203 ;
		:EDATE = 2022204 ;
		:VAR-LIST = "CH4             " ;
		:openmethane_version = "v0.5.4a0" ;

group: emis {
  dimensions:
  	TSTEP = UNLIMITED ; // (1 currently)
  	LAY = 1 ;
  variables:
  	float CH4(TSTEP, LAY, ROW, COL) ;
  	float CH4_UNC(TSTEP, LAY, ROW, COL) ;

  // group attributes:
  		:TDAY = 2LL ;
  data:

   CH4 =
  1.000346, 1.000535, 1.000335, 1.000083, 1,
  1.000088, 1.00026, 1.000097, 1.000013, 1,
  1.000009, 1.000041, 1.000011, 1, 1,
  1, 1, 1, 1, 1,
  1, 1, 1, 1, 1 ;
```

**Comment by prayner** [comment]
For commit a3fbad2 it looks like the change was made in the monthly
but not the daily script, perhaps we want it in both?

**Comment by prayner** [comment]
forget that, I was just landing in an odd place in the commit description

**Review by prayner — APPROVED** [review]

**Pull request merged**